### PR TITLE
Issue 44

### PR DESCRIPTION
**Pull request by Thymas1**
fixes issue 44

### CODE REVIEW

**Comment by robinsandborg** [comment]
To reiterate my previous point about hide-visually. This piece of css is specifically made so that it will be invisible for screens, but still visible for screen readers. This change makes it so that everything that is hidden will still be read by screen readers, and thus the whole expand behaviour is broken for non-seeing users.

See [reach-ui](https://reach.tech/disclosure) for a good, accessible implementation.

File: src/components/Expand/Expand.js [hunk 1]
```diff
@@ -140,6 +138,7 @@ export const ExpandBtn = styled.button`
 export const ExpandItem = styled.div`
   opacity: 0;
   animation: 0.4s ${BounceIn} 0.05s ease-out forwards 1;
+  ${p => (p.isExpanded ? "" : hideVisually)};
```
Comment (robinsandborg) [inline src/components/Expand/Expand.js:141]: This breaks accessibility

**Review by robinsandborg — CHANGES_REQUESTED** [review]
Please improve accessibility for non-seeing users

**Comment by Thymas1** [comment]
Yes, improve accessibility for non-seeing users! Thanks for the input!

**Comment by Thymas1** [comment]
From @robinsandborg check out: https://reach.tech/disclosure/

**Pull request closed without merging**